client/packages/core/src/Reactor.js

```diff
@@ -275,7 +275,9 @@ export default class Reactor {
     this._oauthCallbackResponse = this._oauthLoginInit();
 
     // kick off a request to cache it
-    this.getCurrentUser();
+    this.getCurrentUser().then((userInfo) => {
+      this.syncUserToEndpoint(userInfo.user);
```

Comment:
Since the cookie only lives for 24 hours, should we have something in reactor that periodically refreshes it?

Reply:
I have it sync in the background when reactor gets init'd, not sure of a better solution? ideas welcome